src/test/java/pages/HomePage.java

```diff
@@ -0,0 +1,132 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+
+import java.util.List;
+
+public class HomePage extends BasePage{
+
+    WebElement playlistElement = null;
+
+    @FindBy(css = "input[name='name']")
+    WebElement playlistInputfield;
+
+    @FindBy(css = "div.success.show")
+    WebElement popUpnotification;
+
+
+    @FindBy(css = "i[class='fa fa-plus-circle create']")
+    WebElement playlistPlusBtn;
+
+    @FindBy(css = "[data-testid='playlist-context-menu-create-simple']")
+    WebElement simplePlaylist;
+
+    @FindBy(css = "[class='del btn-delete-playlist']")
+    WebElement deleteBtn;
+
+    @FindBy(css = "button[class='ok']")
+    WebElement okBtn;
+
+    @FindBy(css = "[class='home']")
+    WebElement homePage;
+
+    @FindBy(xpath = "//*[@id='playlists']  //li[@class='playlist playlist']")
+    List<WebElement> playlistLocators;
```

Comment:
If a variable is used only by other methods of that class, it should be declared as private.


I recommend:  make all the locators and WebElements private, because private keyword in java allows most restrictive access to variables and methods and offer strongest form of Encapsulation. 

For example

    ```
    private By shuffleBtnLocator = By.cssSelector(".btn-shuffle-all");
    private By firstSongLocator = By.cssSelector(".play");

```
   @FindBy(css = "img.avatar")
    private WebElement avatar
```